server/chat-plugins/help.ts

```diff
@@ -69,7 +67,7 @@ export class HelpResponder {
 	queue: string[];
 	data: PluginData;
 	constructor(data: PluginData) {
-		this.data = data;
+		this.data = data || {};
```

Comment:
It does not seem like `|| {}` should be happening at all.

Reply:
No, that should be throwing at this point, you're right.